component/otelcol/receiver/kafka/kafka.go

```diff
@@ -81,7 +81,7 @@ var DefaultArguments = Arguments{
 	},
 	HeaderExtraction: HeaderExtraction{
 		ExtractHeaders: false,
-		Headers:        []string{},
+		Headers:        nil,
```

Comment:
The flow docs have a default of `[]` so I think we need to update that with this change.

Reply:
I chatted with Erik offline about this; `[]string{}` and `nil` are identical in River (both are `[]`), so the docs wouldn't need to change. The difference is only meaningful specifically from a Go context when trying to see if the Go values match defaults exactly.